ocaml/idl/datamodel_types.ml

```diff
@@ -179,6 +180,12 @@ let release_order_full = [{
       version_major = 2;
       version_minor = 7;
       branding   = "XenServer 7.2";
+    }; {
+      code_name     = Some rel_inverness;
```

Comment:
Please leave the same version numbers as falcon and use Unreleased for the branding. We'll bump it when we know the actual number.